scripts/tests/run_darwin_framework_ota_test.py

```diff
@@ -111,6 +111,13 @@ def cmd_run(context, darwin_framework_tool, ota_requestor_app, ota_data_file, ot
     if ota_requestor_app is None:
         ota_requestor_app = paths_finder.get('chip-ota-requestor-app')
 
+    if darwin_framework_tool is not None:
+        darwin_framework_tool = SubprocessInfo(kind='tool', path=darwin_framework_tool)
+
+    if ota_requestor_app is not None:
+        ota_requestor_app = SubprocessInfo(kind='app', path=ota_requestor_app
+                                           ).with_args('--otaDownloadPath', ota_destination_file)
```

Comment:
There is no need to use `with_args()` here, just create `SubprocessInfo` with args.